gammapy/estimators/map/tests/test_excess.py

```diff
@@ -142,6 +154,27 @@ def test_significance_map_estimator_map_dataset(simple_dataset):
     assert_allclose(result["npred_excess_ul"].data[0, 10, 10], 107.806275, atol=1e-3)
 
 
+def test_significance_map_estimator_map_dataset_mask_safe(simple_dataset_mask_safe):
+    simple_dataset_mask_safe.exposure = None
+    energy_edges = simple_dataset_mask_safe.counts.geom.axes[0].edges
+    energy_edges = [energy_edges[0], energy_edges[-1]]
+    estimator = ExcessMapEstimator(
+        0.1 * u.deg, selection_optional=["all"], energy_edges=energy_edges
+    )
```

Comment:
```suggestion
    estimator = ExcessMapEstimator(0.1 * u.deg, selection_optional=["all"])
```
If you take min and max edges, you don't need to specify them.